cli/register/register.go

```diff
@@ -361,12 +361,16 @@ func DoIt(org, pattern, nodeIdTok, userPw, inputFile string, nodeOrgFromFlag str
 
 	// Now drop into the long wait for a service to get started on the node.
 	if waitService != "" {
-		msgPrinter.Printf("Horizon node is registered. Workload services should begin executing shortly.")
-		msgPrinter.Println()
-
-		// Wait for the service to be started.
-		WaitForService(waitOrg, waitService, waitTimeout, pattern, pat, nodeType, anaxArch, org, nodeIdTok)
+		if nodeType == persistence.DEVICE_TYPE_CLUSTER {
+			msgPrinter.Printf("-s is not supported for nodeType: %s, Registration will proceed without waiting for service", nodeType)
```

Comment:
Please reword the text: "NOTE: The -s flag is currently not supported for nodeType: %s. Node registration will complete without waiting for the service to start.", nodeType